test/com/esotericsoftware/kryo/io/ByteBufferInputOutputTest.java

```diff
@@ -87,4 +89,36 @@ void testByteBufferOutputPosition () {
 		assertEquals(10, inputBuffer.readInt());
 		assertEquals(9, byteBuffer.position());
 	}
+	
+		@Test
+	void testOverflow () {
```

Comment:
Please use JUnit 5 style assertions for exceptions:

https://stackoverflow.com/a/40268447/441266